recipe/meta.yaml

```
@@ -1,27 +1,27 @@
 {% set name = "deltalake" %}
-{% set version = "0.13.0" %}
+{% set version = "0.14.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deltalake-{{ version }}.tar.gz
-  sha256: e433215eadbc4b845a5b66fc21d8cd18993b4e9fa3fa604b8cd2915271e4e02e
+  sha256: 6aaeac1423c16fa574515f7a0c76b85c4d5fd8e409b0ffcd2c4763f96a6cdb44
 
 build:
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 2
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

IIUC conda-forge configures `pip` by default. I don't think it hurts having the extra arguments, but I don't think they're necessary.

My go-to to understand current best-practice is to check `staged-recipes`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L38

---

Yeah that makes sense, for context this change came from running `grayskull pypi deltalake` and taking the command from there - considering a large chunk of those suggestions aren't correct / necessary, okay with reverting:

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

---

Those arguments should definitely be used and I guess `grayskull` can't assume the recipe will be used in `conda-forge` so it's probably correct for `grayskull` to add them.